plugins/OpenCDM/source/OpenCDMPlugin.cpp

```diff
@@ -183,14 +183,13 @@ bool OpenCDMPlugin::enableTmpOCDMDir(const std::shared_ptr<IDobbyStartState>& st
 
     // on newer builds the OCDM files have moved to a dedicate a /tmp/OCDM
     // directory
-    if ((mkdir(dirPath.c_str(), 0750) != 0) && (errno != EEXIST))
+    if ((mkdir(dirPath.c_str(), 0770) != 0) && (errno != EEXIST))
```

Comment:
I see no issues with the code - except to flag the obvious that it is not good from a security PoV.
(IMO both SVP and OCDM need an overhaul from a security / app isolation PoV).